.buildkite/pipelines/quality-gates/pipeline.tests-qa.yaml

```diff
@@ -47,8 +47,14 @@ steps:
 
   - wait: ~
 
-  - label: ":judge::seedling: Trigger Manual Tests Phase"
+  - label: ":judge::seedling: Trigger Manual Tests Phase - Kibana Release Manager"
     command: "make -C /agent trigger-manual-verification-phase"
     if: build.branch == "main"
     agents:
-      image: "docker.elastic.co/ci-agent-images/manual-verification-agent:0.0.2"
+      image: "docker.elastic.co/ci-agent-images/manual-verification-agent:0.0.3"
+
+  - label: ":judge::seedling: Trigger Manual Tests Phase - Observability"
```

Comment:
I believe the intention is to have a single manual tests step, with the release manager notifying all teams.

@lukeelmers @watson are we tracking who to notify anywhere?

Reply:
:+1:  I'll let watson confirm/wrap up the review.  Changes LGTM otherwise

Reply:
@jbudz yes initially we just wanted a single button only for the release manager. However, based on recent discussion we decided to allow each team that needs to perform manual tests in the QA environment to each have their own button to press. This allows everyone to easily see the progress of the manual tests and means that the release manager does not need to "manually" get verification from each team, but can rely on the status of the buttons as well.